src/book/01-team/05-robocup/03-debriefs.mdx

```diff
@@ -6,6 +6,126 @@ description: Debriefs and takeaways from our performances at RoboCup.
 slug: /team/robocup/debriefs
 ---
 
+## 2022: Bangkok, Thailand
+
+RoboCup Bangkok was the first in-person competition in three years due to COVID-19. All but one team member had never been to an in-person RoboCup competition, and none had been to an overseas RoboCup competition. The team faced further issues when the majority of the luggage did not arrive until the end of the second competition day.
+
+### What went wrong
+
+#### Hardware
+
+- Didn't pack enough cables for battery chargers
+- CM740s are no good, need to get something new. Cannot buy more, and we only have two working.
+- NUC died due to overheating
+- Needed more monitors: one was just not enough
+- Issues of wiring getting in the way – makes it hard to swap things out
+- No soldering happened, but the setup could have been better if we did use it
+- Long ethernet cables became tedious to use and untangle
+- Heat dissipation is an issue
+
+#### Networking
+
+- Issues with connecting to GameController during games
+
+#### Vision
+
+- Red camera. Possible hardware issue? Swapping the cable fixed the issue, but this may have reset the automatic white balance
+- New mesh is too computationally expensive for the NUCs that we have. Potentially go down to one camera per robot and lower the number of images received from the camera
+- The bottom of the mesh, near the robot's feet, bends around, possibly causing issues with ball detection near the feet
+- Hoc was slightly off - odometry or camera calibration
+- NUsight was unreliable for testing vision
+- Theta was in check-in, not carry-on, so couldn’t collect data until later. Did get one image earlier using a theta from CIT Brains
+
+#### NUsight
+
+- Difficulty connecting to NUsight
+
+#### Localisation and Odometry
+
+- No localisation
+- Hoc was slightly off for vision
+
+#### Miscellanious
+
+- No robots until end of first competition day due to missing luggage
+
+### What went well
+
+#### Hardware
+
+- Didn’t replace a single servo at the competition
+- Padding was effective for keeping the robot protected and for colour requirement in inspection
```

Comment:
Would need to check into it again. We did have a nuc overheat but that was because the fan wasn't working during a game. Maybe if we had other solutions on top of that, it would reduce the issue when something did fail. Or just get the robot to monitor heat and do something when it's approaching some sort of critical level.